docs/spec/operators/otel-metrics.md

```diff
@@ -11,31 +11,98 @@ The otel-metrics operator handles collecting and exporting metrics using the
 
 9995
 
-## Global Parameters
+## Parameters
 
-### `otel-metrics-listen`
+### Global Parameters
+
+#### `otel-metrics-listen`
 
 Enables the Prometheus exporter on the address given by `otel-metrics-listen-address` if set to `true`.
 
 Default: `false`
 
-## Instance Parameters
+#### `otel-metrics-listen-address`
 
 The listen address that should be serving Prometheus requests.
 
 Default: `0.0.0.0:2224`
 
-## Instance Parameters
+### Instance Parameters
 
-### `otel-metrics-name`
+#### `otel-metrics-name`
 
 Overrides the name of a datasource and explicitly sets it as name for the export. This is mandatory if you
 want to export the metrics. Use a name that is unique to the gadget+params combination to avoid collision of metrics.
 
-### `otel-metrics-print-interval`
+Fully qualified name: `operator.otel-metrics.otel-metrics-name`
+
+#### `otel-metrics-print-interval`
 
 Interval in which metrics should be emitted as human-readable text. This only has effect for data sources that are
 annotated using `metrics.print=true`. This is also limited to print histograms for now. This functionality might be
 removed in the future.
 
+Fully qualified name: `operator.otel-metrics.otel-metrics-print-interval`
+
 Default: `1000ms`
+
+## Annotations
+
+### Data Source Annotations
+
+#### `metrics.collect`
```

Comment:
IMO, this should be called `metrics.export` as the Otel Metrics operator, if enabled, will always collect, but it will only export if this annotation is set.